fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/TimeMapServiceImpl.java

```diff
@@ -95,7 +94,7 @@ private static final String getLdpcvPath(final String path) {
         if (path.endsWith("/" + LDPCV_TIME_MAP)) {
             return path;
         } else {
-            return Paths.get(path, LDPCV_TIME_MAP).toString();
+            return path + "/" + LDPCV_TIME_MAP;
```

Comment:
From a historical perspective, there is a longer (unresolved) conversation around slashes on containers:
- [Email](https://groups.google.com/forum/#!msg/fedora-tech/OjLO4XOkbPI/ap0JCqMgdiAJ) from @azaroth42
..which resulted in the JIRA [FCREPO-1410](https://jira.duraspace.org/browse/FCREPO-1410)
..which was briefly discussed on a [2015-04-09 Fedora tech call](https://wiki.duraspace.org/display/FF/2015-04-09+-+Fedora+Tech+Meeting).

Reply:
So I guess our options are to either round up a consensus on how to handle trailing slashes (which looks to be a rather heady conversation) or to ignore the issue (and my comment) for the time being. I don't think trailing slash tests are a common occurrence in the code base, so perhaps this PR is not the right time to add them

Reply:
I agree that a 500 just seems wrong,  though.  Do either of you have a suggestion on what a correction would look like?   i.e. what are expected inputs and outputs?  Having never observed `Paths` behaving "as expected" on my platform, I really don't know what behavior this PR is supposed to emulate.

Reply:
@birkland My intent with my misuse of `Paths` was for it to not matter if there was a trailing slash. In the absence of a broader decision about trailing slashes, my opinion would be to preserve the existing behavior, which was that the object would be created whether or not there was a trailing slash (so a 201 response rather than a 404 or a 500). Then we can revisit the general behavior in a more holistic way later.

Reply:
Checked the error, and it seems like the value of `path` ends with `/` whenever the request does.  So the 500 was Modeshape being unhappy with a `//`.  Updated, and added your test @bbpennel